### PR TITLE
L 1475

### PR DESCRIPTION
**Pull request by vsevolod-adrianov**
additional crossPlatformPath method in the class com.epam.jdi.light.common.AdditionalPathUtils
#1475

### CODE REVIEW

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light/src/main/java/com/epam/jdi/light/common/AdditionalPathUtils.java  1
- jdi-light/src/test/java/io/github/epam/properties/ScreensFolderTests.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4646286&bid=15560791)

File: jdi-light/src/test/java/io/github/epam/properties/ScreensFolderTests.java [hunk 1]
```diff
@@ -53,4 +54,30 @@ public void positiveTest(String value) {
         properties.setProperty(SCREENS_FOLDER_PROPERTY.getName(), value);
         validateProperties(properties);
     }
+
+    @Test
+    public void crossPlatformPathTests() {
+        String chromePath = "";
+
+        String os = System.getProperty("os.name");
+        switch (os) {
```
Comment (codacy-review) [inline jdi-light/src/test/java/io/github/epam/properties/ScreensFolderTests.java]: Codacy Issue found: [A switch statement does not contain a break](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4646286)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light/src/main/java/com/epam/jdi/light/common/AdditionalPathUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4646286&bid=15560791)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-light/src/test/java/io/github/epam/properties/CrossPathTests.java  1
- jdi-light/src/main/java/com/epam/jdi/light/common/AdditionalPathUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4646286&bid=15560791)

File: jdi-light/src/test/java/io/github/epam/properties/CrossPathTests.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package io.github.epam.properties;
+
+import com.epam.jdi.light.common.AdditionalPathUtils;
+import org.testng.annotations.Test;
+
+import java.util.Properties;
+
+import static com.epam.jdi.light.common.Property.*;
```
Comment (codacy-review) [inline jdi-light/src/test/java/io/github/epam/properties/CrossPathTests.java]: Codacy Issue found: [Avoid unused imports such as 'com.epam.jdi.light.common.Property'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4646286)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light/src/test/java/io/github/epam/properties/CrossPathTests.java  1
- jdi-light/src/main/java/com/epam/jdi/light/common/AdditionalPathUtils.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4646286&bid=15560791)

**Comment by mf-code** [comment]
[4043] - https://transfer.sh/EmAWl/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4043] - https://transfer.sh/L7oWh/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4043] - https://transfer.sh/ct4GU/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4043] - Allure report on Netlify: https://5df266b5763644964153d69d--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4045] - https://transfer.sh/3VBtK/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4045] - https://transfer.sh/WhRPC/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4045] - https://transfer.sh/10p648/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4045] - Allure report on Netlify: https://5df26d6ede143cfb58a5f430--musing-swartz-6fc060.netlify.com

**Comment by mf-code** [comment]
[4063] - https://transfer.sh/jD91t/jdi-light-html-tests.tar.gz

**Comment by mf-code** [comment]
[4063] - https://transfer.sh/ydjIY/jdi-performance.tar.gz

**Comment by mf-code** [comment]
[4063] - https://transfer.sh/p4leb/jdi-light-examples.tar.gz

**Comment by mf-code** [comment]
[4063] - Allure report on Netlify: https://5df39ed6dbcc44e6b676664e--musing-swartz-6fc060.netlify.com

**Pull request merged**